modules/apps/users-admin/users-admin-impl/src/main/java/com/liferay/users/admin/internal/configuration/settings/UserFileUploadsSettingsImpl.java

```diff
@@ -26,31 +32,65 @@ public class UserFileUploadsSettingsImpl implements UserFileUploadsSettings {
 
 	@Override
 	public int getImageMaxHeight() {
-		return _userFileUploadsConfiguration.imageMaxHeight();
+		UserFileUploadsConfiguration userFileUploadsConfiguration =
+			_getUserFileUploadsConfiguration(CompanyThreadLocal.getCompanyId());
+
+		return userFileUploadsConfiguration.imageMaxHeight();
 	}
 
 	@Override
 	public long getImageMaxSize() {
-		return _userFileUploadsConfiguration.imageMaxSize();
+		UserFileUploadsConfiguration userFileUploadsConfiguration =
+			_getUserFileUploadsConfiguration(CompanyThreadLocal.getCompanyId());
+
+		return userFileUploadsConfiguration.imageMaxSize();
 	}
 
 	@Override
 	public int getImageMaxWidth() {
-		return _userFileUploadsConfiguration.imageMaxWidth();
+		UserFileUploadsConfiguration userFileUploadsConfiguration =
+			_getUserFileUploadsConfiguration(CompanyThreadLocal.getCompanyId());
+
+		return userFileUploadsConfiguration.imageMaxWidth();
 	}
 
 	@Override
 	public boolean isImageCheckToken() {
-		return _userFileUploadsConfiguration.imageCheckToken();
+		UserFileUploadsConfiguration userFileUploadsConfiguration =
+			_getUserFileUploadsConfiguration(CompanyThreadLocal.getCompanyId());
+
+		return userFileUploadsConfiguration.imageCheckToken();
 	}
 
 	@Activate
 	@Modified
 	protected void activate(Map<String, Object> properties) {
-		_userFileUploadsConfiguration = ConfigurableUtil.createConfigurable(
-			UserFileUploadsConfiguration.class, properties);
+		_defaultUserFileUploadsConfiguration =
+			ConfigurableUtil.createConfigurable(
+				UserFileUploadsConfiguration.class, properties);
+	}
+
+	private UserFileUploadsConfiguration _getUserFileUploadsConfiguration(
+		long companyId) {
+
+		try {
+			return _configurationProvider.getCompanyConfiguration(
+				UserFileUploadsConfiguration.class, companyId);
```

Comment:
I think we could use `CompanyThreadLocal.getCompanyId()` directly, no need to put it as a parameter.

Reply:
Fixed it. Thanks